ProcessLib/InitialCondition.h

```diff
@@ -35,7 +35,7 @@ class InitialCondition
 {
 public:
 	virtual ~InitialCondition() = default;
-	virtual double getValue(MeshLib::Node const&) const = 0;
+	virtual double getValue(MeshLib::Node const&, int const) const = 0;
```

Comment:
I oppose: Not resolved. I think, it is also important to have argument names here in the interface.